perf/benchmark/configs/istio/security_request_authn/prerun.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "Configure Security Policies..."
+POLICY_PATH="${WD}/security/generate_policies"
+
+echo "Build Security Policy Generator..."
+go build -o "${LOCAL_OUTPUT_DIR}/generator" "${POLICY_PATH}/generate_policies.go" "${POLICY_PATH}/generate.go" "${POLICY_PATH}/jwt.go"
+
+echo "Apply Security Policy to Cluster..."
+./"${LOCAL_OUTPUT_DIR}/generator" -configFile="${CONFIG_DIR}/security_request_authn/config.json" > "${LOCAL_OUTPUT_DIR}/largeSecurityRequestAuthnPolicy.yaml"
+
+cp "${CONFIG_DIR}/security_request_authn/latency.yaml" "${LOCAL_OUTPUT_DIR}/latency.yaml"
```

Comment:
it seems you are backing up the `latency.yaml` to `LOCAL_OUTPUT_DIR`, replace the environment and override the old one in `CONFIG_DIR`. And in the postrun you revert the backup.

I'm just wondering is it possible to let the test to use the `latency.yaml` in `LOCAL_OUTPUT_DIR` directly so that you don't need the extra cp and revert?

Reply:
For the current test infra, the latency test are using default path of the config file and not configurable. Maybe later, we can make the path configurable.~